### PR TITLE
Minwen Yang_HW01

### PR DESCRIPTION
**Pull request by minwenyang**
For this homework assignment, I added headers, emphasis, links, lists, and an image into the README.md file. I mainly used this [Markdown guide](https://guides.github.com/features/mastering-markdown/) for reference, and it was really helpful. I also watched the workflow tutorial on the course website and learned how to submit the assignments. 

I found it easy to add headers, emphasis, lists, and links. I had some trouble with inserting images. In the guide, they only mentioned how to add images with a link, but I wanted to add an image from my local files. So I looked up for help in [RPubs](https://rpubs.com/RatherBit/90926) and learned to upload image files in GitHub. Afterward, I could insert images in the README file successfully. Another problem I came across was I forgot to pull after I uploaded the image file, so I was unsuccessful with push. After the pull process, everything went well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+* Bake
+* Travel
+
+Here is a picture of me:
+
+![Minwen](IMG_2867.png)
+
+
+## Additional Resource
+Great computing course for new learners of R: [Computing for the Social Science](https://cfss.uchicago.edu/)
```
Comment (djanvier) [inline README.md:22]: 😍

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+* Social Psychology
+
+During my free time, I love to:
+* Paint
+* Bake
+* Travel
+
+Here is a picture of me:
+
+![Minwen](IMG_2867.png)
```
Comment (djanvier) [inline README.md:18]: Nice job using a relative file path!

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        x      |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Reproducibility**         |     x      |              |            |

- Great job!
- Generally, it's easier to make all your changes locally and then submit to GitHub. That way you avoid any merge conflicts or errors.

**Review by elwebster — COMMENTED** [review]
Hi Minwen! Thanks for sharing a bit about yourself. Your file is so neatly organized! You use headers and lists very well to draw the eye to relevant information - I like the inclusion of both professional and personal interests in different lists. You also have a great use of links to MAPSS, the course page, and Peking University -- your page is somehow incredibly informational even though it's a brief introduction, which is wonderful.

The picture is also beautiful, though it seems a bit large compared to the rest of the page - you might consider resizing it in paint or another program. I would also love to see a caption (just so I know where to travel to when COVID restrictions lift).

The file is also reproducible -- I cloned your repository and ran it on my computer, which worked well! 

Overall, this looks really great - lovely to learn a bit about you!

**Pull request closed without merging**